Splitio.Redis/Services/Cache/Classes/RedisEventsCache.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using Splitio.Domain;
+using Splitio.Redis.Services.Cache.Interfaces;
+using Splitio.Services.Shared.Interfaces;
+
+namespace Splitio.Redis.Services.Cache.Classes
+{
+    public class RedisEventsCache : RedisCacheBase, ISimpleCache<Event>
+    {
+        private const string eventKeyPrefix = "events";
+
+        public RedisEventsCache(IRedisAdapter redisAdapter, string machineIP, string sdkVersion, string userPrefix = null)
```

Comment:
@sarrubia should I replicate impressions behavior and use machineIP and sdkVersion for the Redis key? or just use the userPrefix (if any)